src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/LegacyPackageReferenceProject.cs

```diff
@@ -129,6 +129,11 @@ private async Task<bool> IsCentralPackageManagementVersionsEnabledAsync()
             return MSBuildStringUtility.IsTrue(await _vsProjectAdapter.GetPropertyValueAsync(ProjectBuildProperties.ManagePackageVersionsCentrally));
         }
 
+        private async Task<bool> IsCentralPackageVersionOverrideDisabledAsync()
+        {
+            return MSBuildStringUtility.IsFalse(await _vsProjectAdapter.GetPropertyValueAsync(ProjectBuildProperties.EnablePackageVersionOverride));
```

Comment:
Can you split this across multiple lines.  When debugging it can be very useful to inspect the output of `GetPropertyValueAsync`, or to step into `IsFalse` without stepping into `GetPropertyValueAsync`.

Personally, I think that calling a method from inside another method's parameter list should be against our coding guidelines. Variables are free.

Reply:
Yeah sorry, was copy/pasted so I'm going to fix the method above it as well!